internal/releaseversion/releaseversion.go

```diff
@@ -31,11 +32,6 @@ func Validate(ctx context.Context, ctrlCLient client.Client, oldVersion semver.V
 		return microerror.Maskf(releaseNotFoundError, "release %s was not found in this installation", newVersion)
 	}
 
-	// Skip validations for ignored releases.
```

Comment:
This is dangerous, because we would allow ANY upgrade path for clusters running an ignored release. Even downgrading

Reply:
It is meant to be used for development and testing purposes and was never meant for production usage. We can maybe rename the annotation and add `unsupported` in the name.

Reply:
restored